Connecting to multiple AWS accounts (DOC-14) (#33)
openops-cloud/docs: cloud-access/access-levels-permissions.mdx

Connections to **AWS Athena**, **AWS CloudFormation**, and **AWS Compute Optimizer** are configured in the same way as AWS connections.

### Connecting to multiple AWS accounts

> OpenOps supports two approaches for working with multiple AWS accounts.

Reviewer: It's up to the user to decide whether to use separate connections or a single connection with assume-role, it's not necessarily related to their AWS organization structure. Even if the accounts are completely separate, the user may still prefer to go with the assume-role approach.

Author: @maor-rozenfeld Thanks, but I'm not sure what your comment applies to. Do you suggest that I change some of the wording? I've re-read, and I see nothing that implies that choosing one approach over the other is tied to an organization structure.

Reviewer: I think I was referring to the phrasing "If each AWS account is fully independent", you can simply remove it.

> **If each AWS account is fully independent**, or you are not using IAM roles with cross-account trust, you can create a separate AWS connection for each account. In this case, each connection should include that account's access key ID, secret access key, and default region. You can then assign different connections to different workflow actions.

Reviewer: ```suggestion
If you prefer to work with each account separately, you can create an AWS connection for each account. In this case, each connection should include that account's access key ID, secret access key, and default region. You can then assign different connections to different workflow actions.
```

> **If you have a central AWS account that is allowed to assume IAM roles into other accounts** (e.g., a "payer" account managing access across linked "sub-accounts"), you can connect to all of them from a single OpenOps connection. To do this:

Reviewer: ```suggestion
If you want to access multiple accounts together (for example, create a workflow that finds EC2 instances in multiple accounts), you can connect to the accounts from a single OpenOps connection. One account would have to have an assume-role trust with the other accounts. It is common to use the "payer" account in an organization for that purpose. To configure the connection in OpenOps:
```
